src/test/kotlin/com/igorwojda/integer/fizzbuzz/challenge.kt

```diff
@@ -4,7 +4,19 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun fizzBuzz(n: Int): List<String> {
-    TODO("not implemented")
+    val listFizzBuzz = mutableListOf<String>()
+    for (i in 1..n) {
```

Comment:
Can you think of a way of constructing the result list while iterating instead of appending to a mutable list?